usl_pipeline/usl_lib/usl_lib/writers/polygon_writers.py

```diff
@@ -0,0 +1,40 @@
+import typing
+from typing import Tuple
+
+from shapely import geometry
+
+
+def num_list_to_strings(num_list):
+    return list(map(lambda v: str(v), num_list))
+
+
+def write_polygons_to_text_file(
+    polygons_with_masks: list[Tuple[geometry.Polygon, int]],
+    output_file: typing.TextIO,
+    support_mask_values: bool = False,
+) -> None:
+    """Writes polygon information with optional mask values to text file.
+
+    Args:
+        polygons_with_masks: List of tuples combining polygon with associated mask.
+        output_file: File text stream to write to.
+        support_mask_values: Optional indicator that masks should be added into output
+            as additional first column.
+
+    Returns:
+        The generator of tuples combining polygon with associated mask to iterate over.
+    """
+    output_file.write(f"{len(polygons_with_masks)}\n")
+    for polygon_mask in polygons_with_masks:
+        xx, yy = polygon_mask[0].exterior.coords.xy
+        x_list = xx.tolist()
+        y_list = yy.tolist()
```

Comment:
Why do we need the `tolist` calls here rather than iterating over `polygon_mask[0].exterior.coords.xy.xx` & `xy` directly?

Reply:
Yep, that's right, I removed the "tolist" and the "num_list_to_strings" all together.